### PR TITLE
Update skinned mesh improvement mod

### PR DESCRIPTION
**Pull request by Yinsolaya**
Soon as fixy crap ttw.esp is added, please add this.
It was loaded after it due to conflicts with biped models. 
Graphics tag were added due to said biped models.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -1046,6 +1046,18 @@ plugins:
       - crc: 0xCD8AEB60
         util: 'FNVEdit v4.0.4c'
```
Comment (MacSplody) [inline masterlist.yaml]: We can remove the newline here. Since both files are on the same modpage.
```suggestion
```

File: masterlist.yaml [hunk 2]
```diff
@@ -4434,6 +4440,13 @@ plugins:
       - 'GRA WRP Compatibility Patch no Brush Gun.esp'
       - 'GRA-WRP-Patch-Two_Unofficial.esp'
       - 'singleshot_retex.esp'
+      
+        - name: 'PAVE_NV_TTW.esp'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/74786' ]
+    tag: [ Graphics ]
+    after:
+      - 'fixy crap ttw.esp'
+
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
```

Comment (Yinsolaya) [reply masterlist.yaml]: Yeah, we'll save that one for later, haha.

File: masterlist.yaml [hunk 3]
```diff
@@ -1035,16 +1035,22 @@ plugins:
     # 5.4
       - crc: 0xF89DC21D
         util: 'FNVEdit v4.0.4c'
-
-  - name: 'fixy crap ue.esp'
+
```
Comment (MacSplody) [inline masterlist.yaml]: Remove tab.
```suggestion

```

**Comment by MacSplody** [comment]
Thanks.

**Pull request merged**